app/src/main/res/layout/menulayout.xml

```diff
@@ -1,8 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="match_parent">
+    android:layout_height="match_parent"
+    tools:context=".MenuPageActivity"
```

Comment:
Missing 'xmlns:tools' attribute
Source: https://stackoverflow.com/questions/18301389/attribute-is-missing-the-android-namespace-prefix-for-toolscontext